big-dipper/imports/ui/easel_buy/EaselBuy.jsx

```diff
@@ -206,22 +206,33 @@ export default class EaselBuy extends Component {
   };
 
   getNFTDimentions = (nftType, data) => {
```

Comment:
was the spelling supposed to be "getNFTDimensions"?

Reply:
@abdullahjankhan1 Please check and fix